packages/@sanity/dashboard/src/widget.css

```diff
@@ -45,6 +44,7 @@
   min-height: 4em;
   height: 4em;
   margin-top: auto;
+  overflow:hidden;
```

Comment:
Is there no linter in operation which auto-adds a whitespace after `:`?

Reply:
oh. i will add that space manually

Reply:
There is no auto-fix enabled. But stylelint gave it a red line that i missed